gravityflow/gravityflowchecklists
# Update to React 16

This PR updates the version of React to 16 and extracts the repeater component into [gf-repeater](https://www.npmjs.com/gf-repeater) so it can be reused in the Folders extension and perhaps other plugins.

**Testing Instructions**
On master create multiple checklists with varying complex configurations - as many permutations as you can.
When you switch to this branch the configurations should remain identical.
Check that the re-ordering works as expected and that settings remain intact after re-ordering.
